### ISSUE TITLE
Multiplicities 

### ISSUE THREAD

**Issue by negativedensity** [mmajis/plantuml-serverless#4]
Multiplicities are not being handled correctly.

Test case:
```
@startuml
Foo *-- "2" bar
@enduml
```

Result on https://plantuml.nitorio.us:
![39396575-be98f3f8-4af0-11e8-9253-c93d27181566](https://user-images.githubusercontent.com/52726/39397176-5571c232-4afb-11e8-9736-aedf785b54e5.png)

Direct link to nitorio.us case:
https://plantuml.nitorio.us/png/SoWkIImgAStDuNBBprVGqjLLK39IKd1ABE9oICrB0Oe30000

Result on http://plantuml.com/plantuml:
![39396579-d182db14-4af0-11e8-8b17-54d9a882f177](https://user-images.githubusercontent.com/52726/39397180-62a162c8-4afb-11e8-92c1-dcda84543713.png)

Direct link to plantuml.com case:
http://www.plantuml.com/plantuml/uml/SoWkIImgAStDuNBBprVGqjLLK39IKd1ABE9oICrB0Oe30000

**Comment by mmajis** [comment]
Yes, in fact any arrow captions on diagrams other than sequences are not showing up. Looks like I need to find a better way to compile graphviz dot statically with its dependencies...

**Comment by mmajis** [comment]
I’ll try to release a fixed version tomorrow.

**Comment by mmajis** [comment]
Ok @negativedensity, the fix is out in v0.0.5 available in the AWS Serverless Repository here: https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:293246570391:applications~plantuml-render

Let me know if it works for you!

Thanks!
